windows/src/developer/TIKE/compile/CompileKeymanWeb.pas

```diff
@@ -637,6 +642,142 @@ function TCompileKeymanWeb.JavaScript_ContextMatch(fkp: PFILE_KEY; context: PWid
     else Result := JavaScript_CompositeContextValue(fkp, context);
 end;
 
+function TCompileKeymanWeb.JavaScript_Rule(FTabStops, FElse: string; fgp: PFILE_GROUP; fkp: PFILE_KEY): string;
```

Comment:
I'm having trouble finding what is calling this method; how does it integrate into existing code?

Reply:
Lines 739 and 759 call this function